tools/shell/embedded_shell.cpp

```diff
@@ -207,6 +208,14 @@ void EmbeddedShell::run() {
             printNodeSchema(lineStr.substr(shellCommand.SHOW_NODE.length()));
         } else if (lineStr.rfind(shellCommand.SHOW_REL) == 0) {
             printRelSchema(lineStr.substr(shellCommand.SHOW_REL.length()));
+        } else if (lineStr.rfind(shellCommand.LOGGING_LEVEL) == 0) {
+            auto logLevelStr = regex_replace(
```

Comment:
Just use `ltrim` here.
Besides, the processing should be consistent with SHOW_NODE and SHOW_REL.

Reply:
Created a new function, and used ltrim in the new function